dask/dataframe/tests/test_dataframe.py

```diff
@@ -817,8 +817,11 @@ def f(df, partition_info=None):
         assert partition_info is not None
         assert "number" in partition_info
         assert "division" in partition_info
+        assert dsk[("x", partition_info["number"])].equals(df)
+        assert dsk[("x", d.divisions.index(partition_info["division"]))].equals(df)
         return df
 
+    print("in test")
```

Comment:
I think this snuck in accidentally :)

Reply:
Uh-oh. fixed. The style changes remained even after installing `pre-commit`. Removed them manually, but the pre-commit hook runs black (the version I was using before) and reverted all the removals. Not sure if anyone else sees this problem. Can you share your black version? I can just use the same and see if it works.

Reply:
Actually I think the black changes will go away if you rebase off latest master. Are you comfortable trying that?

Reply:
Sorry for the delay. Rebasing wasn't enough because pre-commit would mess it up again. Anyway, I uninstalled pre-commit, fixed it and rebased it.

Reply:
Oops my rebase had unintentional consequences. I rebased with master, but wasn't sure if force push to my branch was allowed. So I tried to rebase and messed up. I'll create a new branch with just my changes